src/widget.js

```diff
@@ -263,6 +263,14 @@ function setDepartment(department) {
 	callHook('setDepartment', department);
 }
 
+function setBusinessUnit(businessUnits) {
+	callHook('setBusinessUnit', businessUnits);
+}
+
+function clearBusinessUnit() {
```

Comment:
I don't know why but the new method is not working in my local env:

![Screen Shot 2022-01-19 at 10 26 11 AM](https://user-images.githubusercontent.com/59577424/150140679-491b1ab7-d150-4f3a-b3ad-8f22d605e6aa.png)

Reply:
I might have a clue about this one. This happened on my local as well & I realised that the changes to this file - `src/widget.js` - doesn't get picked up by live-reload server.

So I had to build the project again using `yarn build && yarn start`. Check if that helps you here

Reply:
It worked. Thanks!